packages/calypso-e2e/src/lib/components/sidebar-component.ts

```diff
@@ -64,25 +63,15 @@ export class SidebarComponent {
 
 		if ( envVariables.VIEWPORT_NAME === 'mobile' ) {
 			await this.openMobileSidebar();
-		} else if ( await this.isSideBarCollapsed() ) {
-			console.info( 'Sidebar is collapsed, expanding...' );
-			await this.toggleSidebar();
 		}
 
 		// Top level menu item selector.
 		const itemSelector = `${ selectors.sidebar } :text-is("${ item }"):visible`;
-
-		if ( envVariables.VIEWPORT_NAME === 'mobile' || ! subitem ) {
-			// when on mobile, or if the main item is the target, click it
-			await this.page.dispatchEvent( itemSelector, 'click' );
-		} else {
-			//  only hover on Desktop when the goal is accessing a subitem
-			await this.page.dispatchEvent( itemSelector, 'mouseover' );
-		}
```

Comment:
I wanted to force a 'single path' for Sidebar interactions instead of supporting two distinct ways. By always ensuring the sidebar is in an expanded state, we are always taking the path of interacting with the full-size sidebar. Combined with the fact that our sidebar implementation had been working reliably for many months now, I felt it was unnecessary to introduce yet another complication following the idea of KISS.

Now, if the argument is that we should also test the hover-access method, then I think it is worth implementing a new method to perform the hover access, but I don't think that should be the main method for our test scripts to interact with the sidebar.

Reply:
Makes sense and sounds good to me! 👍